content/applications/hr/time_off/allocations.rst

```diff
@@ -74,28 +76,60 @@ information:
 - :guilabel:`Allocation`: Enter the amount of time that is being allocated to the employees. This
   field displays the time in either :guilabel:`Hours` or :guilabel:`Days`, depending on how the
   selected :ref:`Time Off Type <time_off/time-off-types>` is configured.
-- :guilabel:`Mode`: Using the drop-down menu, select how the allocation is assigned. This selection
-  determines who receives the time off allocation. The options are :guilabel:`By Employee`,
-  :guilabel:`By Company`, :guilabel:`By Department`, or :guilabel:`By Employee Tag`.
-
-  Depending on what is selected for the :guilabel:`Mode`, the field beneath :guilabel:`Mode` is
-  labeled either: :guilabel:`Employees`, :guilabel:`Company`, :guilabel:`Department`, or
-  :guilabel:`Employee Tag`.
-
-  Using the drop-down menu, indicate the specific employees, company, department, or employee tags
-  receiving this time off.
-
-  Multiple selections can be made for either :guilabel:`Employees` or :guilabel:`Employee Tag`.
-
-  Only one selection can be made for the :guilabel:`Company` or :guilabel:`Department`.
 - :guilabel:`Add a reason...`: If any description or note is necessary to explain the time off
   allocation, enter it in this field at the bottom of the form.
 
 .. image:: allocations/new-allocation.png
-   :align: center
    :alt: A new allocation form with all the fields filled out for the annual two week vacation
          granted to all employees.
 
+Multiple Allocations
+--------------------
+
+When allocating time off, it is common to allocate time to several employees at once. This is done
+using the :guilabel:`Multiple Requests` feature.
+
+To allocate time to multiple employees in a single allocation, navigate to :menuselection:`Time Off
+app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(gear)` icon in the
+upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
```

Comment:
Moving forward, we will refer to this as the "Actions" icon, when using the Actions menu in the UI
```suggestion
app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(Actions)` icon in
the upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
```